### PR TITLE
Change dir structure and added some examples

### PR DESCRIPTION
**Pull request by ramyaksingh98**

### CODE REVIEW

File: golang/araalictl/araalictl.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package main
+package araalictl
```
Comment (araalinetworks) [inline golang/araalictl/araalictl.go:1]: will this break the main.go in this folder? Try running and testing 

go run .

File: golang/araalictl/araalictl.go [hunk 2]
```diff
@@ -177,3 +177,21 @@ func GetLinks(zone, app, tenant string) []Link {
...
+	tenantStr := func() string {
+		if len(tenant) == 0 {
+			return ""
+		}
+		return "-tenant=" + tenant
+	}()
+
+	if clusterName == "" {
+		RunCmd(fmt.Sprintf("/opt/araali/bin/araalictl fortify-k8s %s %s",
+			tenantStr, clusterName))
```
Comment (araalinetworks) [inline golang/araalictl/araalictl.go:192]: you dont need the if else. "" clusterName will not cause any harm

**Review by araalinetworks — APPROVED** [review]
looks good. Saw your changes in main.

File: golang/araalictl/araalictl.go (hunk 1 above)
Comment (ramyaksingh98) [reply golang/araalictl/araalictl.go:1]: I tested the main.go in the parent directory, was working fine

**Pull request merged**